features/favorite.feature

```diff
@@ -0,0 +1,9 @@
+Feature: Favorite
+
+  Scenario: Favorite Creation
+    Given I am logged into the site
+    When I visit a user's profile
+    And I click "Follow"
+    When I visit the "/users/following" page
```

Comment:
Yea, style wise, only the first `When` starts with `When`, then the following ones get `And` until it's a different one. I wrote up a cucumber style guide a few months back. let me see if I can find it.